compiler/rustc_codegen_llvm/src/context.rs

```diff
@@ -54,6 +54,9 @@ pub struct CodegenCx<'ll, 'tcx> {
 
     /// Cache instances of monomorphic and polymorphic items
```

Comment:
IIUC this does *not* contain statics any more. Might be worth a comment.

Why did you split statics into a separate cache?

Reply:
I originally though it was necessary, but am not so sure anymore